src/usb_iss/defs.py

```diff
@@ -5,7 +5,8 @@
 
 # USB-ISS command
 CMD_USB_ISS = 0x5A
-
+VID_USB_ISS = 0x04D8
+PID_USB_ISS = 0xFFEE

```

Comment:
Nit: these are unrelated to `CMD_USB_ISS`, so probably deserve their own section.